bin/convert-speaker-images.js

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env node
+
+var argv = require('minimist')(process.argv.slice(2))
+    , lwip = require('lwip')
+    , path = require('path')
+
+function print(msg) {
+    console.log(msg + '\n')
+} 
+
+function usage() {
+    print('Usage:')
+    print('npm run speakerimages -- --imagepath=<path/to/file/name.jpeg>')
+}
+
+// main
+if (!argv.imagepath) {
+    print('Missing script arguments')
+    usage()
+} else {
+    var filePath = path.resolve(__dirname + '/../' + argv.imagepath)
+        , pathParts = filePath.split('/')
+        // everything but the file name
+        , pathBase = pathParts.slice(0, -1)
+        , fileName = pathParts.slice(-1)
+        , convertPath = path.resolve(pathBase.join('/') + '/' + fileName[0].split('.')[0] + '.jpg')
```

Comment:
uhh, yeah I was talking about the Unix slash delimiters...I don't think normalize takes care of that